I tried some things but the error remains. I can reproduce this locally so I don't think it's a Travis issue. 

<details>
<summary>Output</summary>

```

Preparing transaction: ...working... done
Verifying transaction: ...working... failed
Build prefix failed with prefix length 255
WARNING:conda_build.environ:Build prefix failed with prefix length 255
Error was: 
WARNING:conda_build.environ:Error was: 
Placeholder of length '80' too short in package /home/jaime/.local/anaconda/conda-bld/yank-examples_1559042337128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/bin/acdoctor.
The package must be rebuilt with conda-build > 2.0.
WARNING:conda_build.environ:Placeholder of length '80' too short in package /home/jaime/.local/anaconda/conda-bld/yank-examples_1559042337128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/bin/acdoctor.
The package must be rebuilt with conda-build > 2.0.
One or more of your package dependencies needs to be rebuilt with a longer prefix length.
WARNING:conda_build.environ:One or more of your package dependencies needs to be rebuilt with a longer prefix length.
Falling back to legacy prefix length of 80 characters.
WARNING:conda_build.environ:Falling back to legacy prefix length of 80 characters.
Your package will not install into prefixes > 80 characters.
WARNING:conda_build.environ:Your package will not install into prefixes > 80 characters.
Traceback (most recent call last):
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/path_actions.py", line 423, in verify
    self.file_mode)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/portability.py", line 59, in update_prefix
    update_file_in_place_as_binary(realpath(path), _update_prefix)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/gateways/disk/update.py", line 42, in update_file_in_place_as_binary
    fh.write(callback(data))
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/portability.py", line 41, in _update_prefix
    data = replace_prefix(mode, original_data, placeholder, new_prefix)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/portability.py", line 66, in replace_prefix
    data = binary_replace(data, placeholder.encode('utf-8'), new_prefix.encode('utf-8'))
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/portability.py", line 95, in binary_replace
    data = pat.sub(replace, data)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/portability.py", line 90, in replace
    raise _PaddingError
conda.core.portability._PaddingError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/environ.py", line 865, in create_env
    execute_actions(actions, index)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/plan.py", line 320, in execute_actions
    execute_instructions(plan, index, verbose)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/plan.py", line 532, in execute_instructions
    cmd(state, arg)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/instructions.py", line 73, in UNLINKLINKTRANSACTION_CMD
    unlink_link_transaction.execute()
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/link.py", line 236, in execute
    self.verify()
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/link.py", line 223, in verify
    exceptions = self._verify(self.prefix_setups, self.prefix_action_groups)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/link.py", line 522, in _verify
    for target_prefix, prefix_group in iteritems(prefix_action_groups)),
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/link.py", line 518, in <genexpr>
    exceptions = tuple(exc for exc in concatv(
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/link.py", line 346, in _verify_individual_level
    error_result = axn.verify()
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/core/path_actions.py", line 426, in verify
    len(self.prefix_placeholder))
conda.exceptions.PaddingError: Placeholder of length '80' too short in package /home/jaime/.local/anaconda/conda-bld/yank-examples_1559042337128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/bin/acdoctor.
The package must be rebuilt with conda-build > 2.0.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jaime/.local/anaconda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 456, in main
    execute(sys.argv[1:])
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 447, in execute
    verify=args.verify, variants=args.variants)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/api.py", line 208, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/build.py", line 2314, in build_tree
    notest=notest,
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/build.py", line 1397, in build
    create_build_envs(m, notest)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/build.py", line 1285, in create_build_envs
    is_cross=m.is_cross, is_conda=m.name() == 'conda')
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/environ.py", line 890, in create_env
    clear_cache=clear_cache, is_cross=is_cross)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda_build/environ.py", line 859, in create_env
    display_actions(actions, index)
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/exports.py", line 229, in display_actions
    actions['LINK'] = [index[d] for d in actions['LINK']]
  File "/home/jaime/.local/anaconda/lib/python3.7/site-packages/conda/exports.py", line 229, in <listcomp>
    actions['LINK'] = [index[d] for d in actions['LINK']]
KeyError: PackageRecord(_hash=-7655569569748998813, name='ca-certificates', version='2019.3.9', build='hecc5488_0', build_number=0, channel=Channel("conda-forge/linux-64"), subdir='linux-64', fn='ca-certificates-2019.3.9-hecc5488_0.tar.bz2', md5='ab48532bfb7c5709ed3145ae25c69353', url='https://conda.anaconda.org/conda-forge/linux-64/ca-certificates-2019.3.9-hecc5488_0.tar.bz2', arch=None, platform=None, depends=(), license='ISC', timestamp=1552136288.257, size=149766)

```

</details>

The key line is :
```
One or more of your package dependencies needs to be rebuilt with a longer prefix length.
```

Since the recipe only specifies `yank` and `ambermini`... yep, it's `ambermini`. Should we unpin this dependency? Or at least, make it less strict (greater than)?

--- a/.travis.yml
+++ b/.travis.yml
@@ -30,18 +30,17 @@ install:
   # Activate the test environment
   - conda activate test
   # Build the recipe
-  - conda build devtools/conda-recipe
+  - conda build devtools/conda-recipe --no-test
   # Install the package
   - conda install --use-local ${PACKAGENAME}
   # Install testing dependencies
-  - conda install --quiet nose nose-timer pytest
+  - conda install --quiet pytest
   # Install OpenEye dependencies
   - pip install -i https://pypi.anaconda.org/OpenEye/simple OpenEye-toolkits && python -c "import openeye; print(openeye.__version__)"
+
+  script:
   # Test the package
-  - cd tests
-  - pytest test_syntax.py
-  - nosetests --nocapture --verbosity=2 --with-timer -a '!slow' test_examples.py
-  - cd ..
+  - pytest -m 'not slow'
 
 env:
   matrix:

--- a/devtools/conda-recipe/meta.yaml
+++ b/devtools/conda-recipe/meta.yaml
@@ -8,18 +8,10 @@ source:
 requirements:
   build:
     - yank >=0.17.0
-    - ambermini 16.16.0 6
 
   run:
     - yank >=0.17.0
-    - ambermini 16.16.0 6
 
-test:
-  requires:
-    - nose
-    - nose-timer
-  imports:
-    - yank
 
 about:
   home: https://github.com/choderalab/yank-examples

--- a/pytest.ini
+++ b/pytest.ini
@@ -1,0 +1,3 @@
+[pytest]
+markers =
+    slow: marks tests as slow (deselect with '-m "not slow"')

--- a/tests/test_examples.py
+++ b/tests/test_examples.py
@@ -21,8 +21,7 @@ import re
 
 import openmoltools as omt
 
-from nose.plugins.attrib import attr
-from unittest import skipIf
+import pytest
 
 import yank
 
@@ -163,7 +162,7 @@ def run_examples(example_directory, yaml_name):
     return
 
 
-@attr('slow')
+@pytest.mark.slow
 def test_t4_p_xylene_explicit():
     """
     Test p-xylene binding to t4 lysozyme example in explicit solvent
@@ -173,7 +172,7 @@ def test_t4_p_xylene_explicit():
     run_examples(example_dir, yaml_name)
 
 
-#@attr('slow')
+@pytest.mark.slow
 def test_t4_p_xylene_implicit():
     """
     Test p-xylene binding to t4 lysozyme example in implicit solvent
@@ -183,8 +182,8 @@ def test_t4_p_xylene_implicit():
     run_examples(example_dir, yaml_name)
 
 
-@attr('slow')
-@skipIf(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
+@pytest.mark.slow
+@pytest.mark.skipif(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
 def test_t4_all_ligands_explicit():
     """
     Test that the all-binders to t4 lysozyme example works in explicit solvent
@@ -195,8 +194,8 @@ def test_t4_all_ligands_explicit():
     run_examples(example_dir, yaml_name)
 
 
-#@attr('slow')
-@skipIf(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
+@pytest.mark.slow
+@pytest.mark.skipif(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
 def test_t4_all_ligands_implicit():
     """
     Test that the all-binders to t4 lysozyme example works in implicit solvent
@@ -207,7 +206,7 @@ def test_t4_all_ligands_implicit():
     run_examples(example_dir, yaml_name)
 
 
-@attr('slow')
+@pytest.mark.slow
 def test_hydration_phenol_implicit():
     """
     Test that the phenol hydration implicit example
@@ -217,7 +216,7 @@ def test_hydration_phenol_implicit():
     run_examples(example_dir, yaml_name)
 
 
-@attr('slow')
+@pytest.mark.slow
 def test_hydration_phenol_explicit():
     """
     Test that the phenol hydration explicit example
@@ -227,8 +226,8 @@ def test_hydration_phenol_explicit():
     run_examples(example_dir, yaml_name)
 
 
-#@attr('slow')
-@skipIf(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
+@pytest.mark.slow
+@pytest.mark.skipif(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
 def test_binding_host_guest():
     """
     Test the host guest binding example
@@ -238,8 +237,8 @@ def test_binding_host_guest():
     run_examples(example_dir, yaml_name)
 
 
-#@attr('slow')
-@skipIf(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
+@pytest.mark.slow
+@pytest.mark.skipif(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
 def test_hydration_freesolv():
     """
     Test that the hydration freesolv database example works

--- a/tests/test_files_exist.py
+++ b/tests/test_files_exist.py
@@ -1,0 +1,47 @@
+#!/usr/bin/python
+
+# =============================================================================================
+# MODULE DOCSTRING
+# =============================================================================================
+
+"""
+Test YAML syntax in examples.
+
+"""
+
+# =============================================================================================
+# GLOBAL IMPORTS
+# =============================================================================================
+
+from pathlib import Path
+import filecmp
+import os
+import sys
+
+import pytest
+import yaml
+
+
+def _dirs_equal(comparer):
+    """
+    Recurse over subdirs testing that there are no:
+    - files with same name, but different contents
+    - files with different names
+    """
+    yield all(not bool(x) for x in (comparer.diff_files, comparer.left_only, comparer.right_only))
+    for name, subdir in comparer.subdirs.items():
+        print(name)
+        yield from _dirs_equal(subdir)
+
+
+def test_files_exist():
+    """
+    Compare examples copied to $PREFIX/share/yank
+    and those available in the current repo. They should match!
+    """
+    repo_examples = Path(__file__).parent.parent / 'examples'
+    prefix_examples = Path(sys.prefix) / 'share' / 'yank' / 'examples'
+    assert repo_examples.is_dir()
+    assert prefix_examples.is_dir()
+    comparer = filecmp.dircmp(repo_examples, prefix_examples)
+    assert all(_dirs_equal(comparer))

